### PR TITLE
Add new product

### PR DESCRIPTION
**Pull request by tiennguyen98**

### CODE REVIEW

**Comment by tiennguyen98** [comment]
ready

**Comment by tiennguyen98** [comment]
ready

**Comment by BunnyPi04** [comment]
commented

**Comment by tiennguyen98** [comment]
ready

File: app/Http/Controllers/Client/ProductController.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+use App\Http\Requests\ProductRequest;
+
+class ProductController extends Controller
+{
+    public function postProduct()
+    {
+        $categories = Category::convertToArray();
+        $cities = City::convertToArray();
+        
+
```
Comment (BunnyPi04) [inline app/Http/Controllers/Client/ProductController.php]: dư 1 dòng trắng rồi

File: public/js/newproduct.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        'X-Requested-With': 'XMLHttpRequest',
+    },
+    init: function () {
+        var myDropzone = this;
+        $('.newpost__save').click(function(){
+            myDropzone.processQueue();
+        });
+    },
+    success: function (file, done) {
+        if(done.error == undefined){
```
Comment (BunnyPi04) [inline public/js/newproduct.js]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by tiennguyen98** [comment]
ready

File: resources/views/client/product/new.blade.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                ]) !!} 
+
+                {!! Form::hidden('image_data', null, ['id' => 'imageData']) !!}
+                <div class="form-group mt-4 row">
+                    <div class="input-group mb-2 col-sm-8">
+                        <div class="input-group-prepend">
+                            <div class="input-group-text">@lang('admin.product.name')</div>
+                        </div>
+                        {!! Form::text('name', null, 
+                                [
```
Comment (BunnyPi04) [inline resources/views/client/product/new.blade.php]: tab lại

**Comment by BunnyPi04** [comment]
commented
conflict rồi nhé

**Comment by tiennguyen98** [comment]
ready

**Pull request merged**